main.py

```diff
@@ -534,52 +534,80 @@ async def list_result_directory() -> list[int]:
 
 
 @app.get("/dataset/master/")
-async def get_master_dataset(key: str) -> tuple[float, Union[int, float, list]]:
-    """Returns the current timestamp and the dataset broadcast to artiq master.
+async def get_master_dataset(key: str) -> Union[int, float, list]:
+    """Returns the dataset broadcast to artiq master.
 
     Args:
         key: The key of the target dataset.
 
     Returns:
-        If the dataset is not initialized or does not exist, it returns (-1, ()).
+        If the dataset is not initialized or does not exist, it returns an empty tuple.
```

Comment:
Umm.. I think the previous docstring was correct since `DatasetTracker.get()` might return `(-1, ())`.
Instead, the type hinting in terms of this function looks wrong: `Union[int, float, list, tuple]`.

Reply:
Oh I thought FastAPI automatically casts tuples to lists because of the return type annotation.
However, I'm not sure about that..
I'm okay with updating the type hint.